genref/output/md/kubeadm-config.v1beta2.md

```diff
@@ -955,7 +949,7 @@ DNS defines the DNS add-on that should be used in the cluster
 - [DNS](#kubeadm-k8s-io-v1beta2-DNS)
 
 
-DNSAddOnType defines string identifying DNS add-on types.
+DNSAddOnType defines string identifying DNS add-on types
```

Comment:
@tengqm , Is this change coming from the upstream source code?

Reply:
It is a minor issue. The formatting is not consistent.

Reply:
Yes. It is from source code change.